src/lighthouse/cleanup.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+const CI_ROOT = '/home/runner/work/cfgov-lighthouse/cfgov-lighthouse/';
+
+/**
+ * Takes a summary report and deletes all "non-representative" runs from
+ * the filesystem to save disk space.
+ * @param {object} summaryReport A LighthouseSummaryReport (see reports.js)
+ */
+const cleanUpRuns = summaryReport => {
+
+  summaryReport.nonRepresentativeRuns.forEach( run => {
+
+    const filePath = path.join( __dirname, '../../', run.jsonPath.replace( CI_ROOT, '' ) );
+    const fileName = filePath.split( '/' ).pop();
+
+    fs.unlink( filePath, err => {
+      if ( err ) {
+        return console.error( `Tried to delete ${ fileName } but failed. It might already be deleted.` );
+      }
+      console.log( `Deleted non-representative run ${ fileName }.` );
```

Comment:
Might need `// eslint-disable-next-line no-console`

Reply:
I threw a [comment](https://github.com/cfpb/cfgov-lighthouse/pull/6/files/82b8341e09f69f5802a829440a6046cede037563#diff-952fcf8c843809b9c8a5f8a15e9bef5eR1) at the top of the file so it should be okay.